File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    const sorted_sizes = sizes.map(item => item.sort((a,b) => b - a))
    let width = 0
    let height = 0
    for (const [card_width,card_height] of sorted_sizes){
        width = Math.max(width,card_width)
        height = Math.max(height,card_height)
    }
    return width * height
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<vector<int>> sizes) {
    int width = 0;
    int height = 0;
    for (int i = 0; i < sizes.size(); i++){
        int currWidth = sizes[i][0] > sizes[i][1] ? sizes[i][0] : sizes[i][1];
        int currHeight = sizes[i][0] > sizes[i][1] ? sizes[i][1] : sizes[i][0];
        if (width < currWidth) width = currWidth;
        if (height < currHeight) height = currHeight;
    }
    return width * height;
}